### ISSUE TITLE
oxlint.config.mts file not detected

### ISSUE THREAD

**Issue by bbrzoska** [oxc-project/zed-oxc#149]
### Confirm that this is a Bug for the editor extension, not the CLI

- [x] This bug does not happen in the Oxlint/Oxfmt CLI
- [x] After restarting the editor, the bug still appears

### Versions used

```
Zed Extension v0.4.4
oxlint is enabled (v1.56.0)

Operating System and Version:
Node Version: v24.13.1
Node Manager: prototools
```

### What happened?

oxlint.config.mts is not auto-detected by oxc-zed. The config gets picked up just fine in CLI though. Workaround is to set `"oxc.configPath": "./oxlint.config.mts"` manually.

Same issue as https://github.com/oxc-project/oxc/issues/20715

### Reproduction repo

1. Create "./oxlint.config.mts" in repo.
2. Open a file with linting errors that get picked up by CLI - they don't get picked up by the Zed plugin
